tests/unit/synapseclient/unit_test_client.py

```diff
@@ -263,6 +263,12 @@ def setup(self):
             'etag': 'etag',
             'name': 'entity name'
         }
+        self.docker = {
+            'versionNumber': 7,
+            'id': 'syn1009',
+            'etag': 'etag',
+            'name': 'entity name'
+        }
```

Comment:
docker should have an `repositoryName` -- this is used in the code. Is it possible for a docker entity to not have this field? If the backend says no, then the client should throw an exception.